charts/azad-kube-proxy/templates/pdb.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: policy/v1beta1
```

Comment:
Should we have an if statement here and default to it being off?

Reply:
Since azad is so important for the cluster to function as it should, I don't see why. It should always have a minimum of 2 replicas and 1 should always be online. But yes this forces you more or less to have 2 nodes in your cluster so I can add a if statement for it.